src/components/form/InviteeForm.js

```diff
@@ -0,0 +1,93 @@
+import React, {useState} from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Paper from '@material-ui/core/Paper';
+import PartnerDataTextField from './InviteeDataTextField';
+import InviterSelect from './InviterSelect';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+      width: '100%',
+      minWidth: 100,
+    },
+  },
+
+  button: {
+    backgroundColor: '#457b9d',
+    color: '#ffffff',
+    '&:hover': {
+      background: '#1d3557',
+    },
+  },
+}));
+
+export default function InviteeForm(props) {
+  const classes = useStyles();
+
+  const [partnerState, setPartnerState] = useState({
+    name: '',
+    email: '',
+    isGoogler: false,
+  });
+
+  // function: String => (Value => ())
+  const changePartnerState = (key) => {
+    return (newValue) => {
+      if (!(key in partnerState)) {
+        throw 'Key "${key}" is unknown for partner state.';
```

Comment:
./src/components/form/InviteeForm.js
  Line 40:9:   Expected an error object to be thrown  no-throw-literal
  Line 40:15:  Unexpected template string expression  no-template-curly-in-string

Reply:
sorry, should be ``` instead of `'`

Reply:
I have created [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) object with this message. Should be working now without warnings.